src/Tribe/Tickets_Handler.php

```diff
@@ -1043,15 +1046,32 @@ public function ajax_handler_save_settings() {
 
 		if ( ! empty( $params['tribe_ticket_header_image_id'] ) ) {
 			update_post_meta( $id, '_tribe_ticket_header', $params['tribe_ticket_header_image_id'] );
-			wp_send_json_success( $params );
 		} else {
 			delete_post_meta( $id, '_tribe_ticket_header' );
-			wp_send_json_success( $params );
 		}
 
-		// #TODO: placeholder
+		// We reversed this logic on the back end
+		if ( ! empty( $params['tribe_show_attendees'] ) ) {
+			delete_post_meta( $id, '_tribe_show_attendees' );
+		} else {
+			update_post_meta( $id, '_tribe_show_attendees', 1 );
+		}
+
+
+		if ( ! empty( $params['default_ticket_provider'] ) ) {
+			update_post_meta( $id, '_default_ticket_provider', $params['default_ticket_provider'] );
```

Comment:
Is this information that we are saving? If yes, let's go with a `_tribe` prefix.